### PR TITLE
fix: Add organisation user to case assignment request as it is needed…

### PR DESCRIPTION
**Pull request by tomaszpowroznik**

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseRoleService.java [hunk 1]
```diff
@@ -92,9 +93,15 @@ private void grantCaseAssignmentAccess(String caseId, Set<String> users, Set<Cas
...
             final String serviceToken = authTokenGenerator.generate();
+
+            final String organisationId = organisationService.findOrganisation()
+                .map(Organisation::getOrganisationIdentifier)
+                .orElse(null);
+
             List<CaseAssignedUserRoleWithOrganisation> caseAssignedRoles = users.stream()
                 .map(user -> CaseAssignedUserRoleWithOrganisation.builder()
                     .caseDataId(caseId)
+                    .organisationId(organisationId)
```
Comment (nkoranne) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseRoleService.java:104]: as per agreement with CCD : RDM-9557
No organisation ID field in your requests - Org ID fields will be incorporated later on with agreement with CCD,

Comment (nkoranne) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseRoleService.java:104]: Do we need to inform CCD before deploying this change?

Comment (tomaszpowroznik) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseRoleService.java:104]: thanks for pointing that, contacted with Mutlu to confirm if we can use it

Comment (tomaszpowroznik) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseRoleService.java:104]: had chat with Mutlu and Sam about this one, we have green light to

**Review by nkoranne — APPROVED** [review]

**Review by samuelzerozero — APPROVED** [review]

**Review by nkoranne — APPROVED** [review]

**Pull request merged**